File: dinner/Dinner/leap_year_count.py
year = 2025;
sum = 0
for days in range(1900 , year + 1):
	if days % 4 == 0 and days % 100 != 0:
		sum += 1
		



	elif days % 4 == 0 and days % 100 == 0 and days % 400 == 0:
		sum += 1
		


	elif days % 4 == 0 and days % 100 == 0 and days % 400 != 0:
		sum += 1
		
print(sum)

File: dinner/Dinner/LeapYearCount.java
public class LeapYearCount{

public static void main(String... args){
 
int year = 0;
int sum = 0

for(year = 1900; year <= 2025; year++){

if(year % 4 == 0 && year % 100 != 0){
sum += 1;


}

else if(year % 4 == 0 && year % 100 == 0 && year % 400 == 0){
sum += 1;


}         

else if(year % 4 == 0 && year % 100 == 0 && year % 400 != 0){
sum += 1;


}


}

System.out.println(sum);
}
}